src/adr/0071-objectstorage-sse-c.md

```diff
@@ -0,0 +1,50 @@
+# 71. SSE-C for S3 Object Storage
+
+Date: 2024-11-15
+
+## Status
+
+Accepted (lazy consensus)
+
+## Context
+
+To enhance data security for users, Apache James currently supports client-side encryption through AESBlobStoreDAO. However, this solution consumes significant CPU resources for encryption and decryption, impacting overall system performance.
+
+To provide a more efficient encryption option, the team has decided to integrate Server-Side Encryption with Customer-provided keys (SSE-C) for S3 Object Storage. This will allow for enhanced security with optimized performance. SSE-C provides high-level security without the CPU overhead experienced with client-side encryption in James.
```

Comment:
"the team" have no meaning in an ASF context. Prefer using "we".

Reply:
I expect the context to explain in ~1 line what is SSEC.

> the team has decided

This is the decision. Move it.

>  This will allow ...

And this repeats impact